client/dashboard/sites/reset-site-modal/style.scss

```diff
@@ -0,0 +1,4 @@
+.reset-site-modal-progress-bar {
+    width: 100% !important;
+    height: 2px !important;
```

Comment:
Curious, why? The current Core's default 1.5px seems good enough 🤔

Reply:
I found it pretty small especially compared to `4px` in the current dashboard. It was kind of looking like a modern tab-like underline until the progress updated.

| Default | `2px` |
|--------|--------|
| <img width="418" alt="Screenshot 2025-05-28 at 11 52 05 AM" src="https://github.com/user-attachments/assets/fd6707b4-2527-4e4d-9845-c812d355a547" /> | <img width="433" alt="Screenshot 2025-05-28 at 11 52 14 AM" src="https://github.com/user-attachments/assets/5fec7812-9a41-40f0-a222-8c54d9738c46" /> | 

@jasmussen Thoughts?

Reply:
We definitely do not want ad-hoc overrides to any of the components like this, that's pretty key. The 1.5px is intentional, a bit of a branding choice to have a width that matches exactly that of the default focus style, and the default stroke-width of all iconography. 2px is not used as a value anywhere, except for one of the smaller corner radiuses. 

In this case I also think the existing size is fine. Your argument isn't invalid, but given the context: a modal you can close and all is fine, the prominence is correct.

That being said, a 4px version is being proposed here: dhW7zYBamXJIeJIE3wy5f4-fi-111_515. It isn't built yet, and as noted I don't think it's necessary for this specific context, but it is likely to become an option.